nomos-cli/src/cmds/disseminate/mod.rs

```diff
@@ -47,6 +48,7 @@ impl Disseminate {
         let output = self.output.clone();
         let (payload_sender, payload_rx) = tokio::sync::mpsc::unbounded_channel();
         payload_sender.send(bytes).unwrap();
+        let private_key = PrivateKey::default(); // TODO: pass private_key via the cli args.
```

Comment:
Uhm, what is this key used for? Shouldn't be the node itself the one to actually sign stuff?

Reply:
I've used this code to discuss an idea on how blob structure should look like, I'll summarize the discussion in the document. In short, this was added as proposal for blob validation during dissemination (before cert is created), but the idea was discarded.